Common/UnitDefinitions/ElectricApparentEnergy.json

```diff
@@ -11,6 +11,11 @@
     {
       "SingularName": "VoltampereHour",
       "PluralName": "VoltampereHours",
+      "BaseUnits": {
+        "L": "Meter",
+        "M": "Kilogram",
+        "T": "Second"
```

Comment:
> Personally, I don't think that anyone should use the As(UnitSystem) as they are right now

I kind of agree. I believe @tmilnthorp has some insights here, but in my mind I don't yet see a compelling use case for the `UnitSystem` conversion methods and ctors - at least the simple ways I use this library, I have never needed it. 

But having definitions for SI base units seems useful and should allow us to infer stuff down the road.

Reply:
> I would imagine if I wrote an application, like Blender or Solidworks, I would have the user pick the unit of choice for known quantities like Length, Mass, and so on. This would affect how units were displayed, as well as inputs to set values.
> 
> I would then offer presets, something like:
> 
>     * `Metric [cm]`: Length = cm, Mass = kg, Time = s
> 
>     * `Metric [m]`: Length = m, Mass = kg, Time = s
> 
>     * `Imperial [ft]`: Length = ft, Mass = lbs, Time = s
> 
>     * `Imperial [in]`: Length = in, Mass = lbs, Time = s
> 
> 
> And this would all be configurable.
> 

Right, this was my original idea back in #709 where I was thinking about having the `QuantityInfo` / `UnitInfo` include a dictionary of default `UnitSystem` associations / configurations - which contain the `DefaultUnit` (resolving the possible duplicates) etc.. I was thinking of maybe re-opening this for `release-v9` 😄

Reply:
> > I would imagine if I wrote an application, like Blender or Solidworks, I would have the user pick the unit of choice for known quantities like Length, Mass, and so on. This would affect how units were displayed, as well as inputs to set values.
> > I would then offer presets, something like:
> > ```
> > * `Metric [cm]`: Length = cm, Mass = kg, Time = s
> > 
> > * `Metric [m]`: Length = m, Mass = kg, Time = s
> > 
> > * `Imperial [ft]`: Length = ft, Mass = lbs, Time = s
> > 
> > * `Imperial [in]`: Length = in, Mass = lbs, Time = s
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > And this would all be configurable.
> 
> Right, this was my original idea back in #709 where I was thinking about having the `QuantityInfo` / `UnitInfo` include a dictionary of default `UnitSystem` associations / configurations - which contain the `DefaultUnit` (resolving the possible duplicates) etc.. I was thinking of maybe re-opening this for `release-v7` 😄

Reply:
I've removed the `BaseUnits` from the `ElectricApparentEnergy.VoltampereHour` and added "See about adding an SI unit (VoltampereSecond, Joules?)" as the [Skip] reason (similar to my previous comment/question in #1444 regarding the `ElectricReactiveEnergy`).
https://github.com/angularsen/UnitsNet/pull/1473/commits/93e940c89afb22349747bf4a05f375ef4ee802ec

Reply:
Alright 👍